packages/e2e-tests/src/specs/editor/publishingFlow.js

```diff
@@ -19,7 +19,6 @@
  */
 import {
   createNewStory,
-  addRequestInterception,
```

Comment:
Are there any other usages of `addRequestInterception`? Should we remove the file?

Reply:
There shouldn't be.

Might be a good idea to remove it, yeah 👍